src/app/shared/models/article-list-config.model.ts

```diff
@@ -1,11 +1,11 @@
-export class ArticleListConfig {
-  type = 'all';
+export interface ArticleListConfig {
+  type: string;
```

Comment:
hmm...where did the initialization of type='all' go to?

Reply:
whops, apparently I overlooked that :D

Reply:
fixed by initialising the objects with `type: 'all'`